app/controllers/users_controller.rb

```diff
@@ -0,0 +1,27 @@
+class UsersController < ApplicationController
+  http_basic_authenticate_with name: "admin", password: "123"
+  
+  def index
+    current_user
+    @users = User.all.order(id: :asc)
+  end
+
+  def create
+    @user = User.where(username: params[:username], password: params[:password]).first
```

Comment:
No, need to define instance variable( `@` ) if you are using that variable in same method.